lib/mac-token.js

```diff
@@ -91,15 +94,15 @@ MACTokenGenerator.prototype.validate = function(req) {
     return null;
   }
 
-  debug('MAC authorization: %s', authorizationHeader);
+  debug(g.f('MAC authorization: %s', authorizationHeader));
```

Comment:
Actually, you don't have to globalize debug messages, so you could just revert this file.